smart-home-provider/cloud/auth-provider.js

```diff
@@ -304,45 +304,49 @@ Auth.registerAuth = function(app) {
 
 /**
  * Decode authorization into client_id and client_secret.
+ * @param {Object} req ExpressJS request object
  * @return {{}}
  * {
  *   client_id: "CLIENT_ID",
  *   client_secret: "CLIENT_SECRET"
  * }
  */
 function getHeaderAuthorization(req){
-  let headerAuthor = {};
-    try{
-      if(!!req.headers.authorization){
-        let authorizationString = Buffer.from(req.headers.authorization.split(" ")[1], 'base64').toString();
-        if(!!authorizationString){
-          let authorizationArray = authorizationString.split(':');
-          headerAuthor = {
-            client_id: authorizationArray[0],
-            client_secret: authorizationArray[1]
-          }
-        }
-      }
-    }catch(e){
+  try{
+    if(!req.headers.authorization)
```

Comment:
If your `if` statement is broken into two lines, it should have braces: https://google.github.io/styleguide/jsguide.html#formatting-braces-all